mne/utils/docs.py

```diff
@@ -184,7 +184,7 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 to change (even if the function is only applied to channels in ``picks``).{}
 
 .. note:: If ``n_jobs`` > 1, more memory is required as
-          ``len(picks) * n_times`` additional time points need to
+          ``n_jobs * n_times`` additional time points need to
```

Comment:
are you certain that this change is correct?  The original text implies that *full copies of the array* are made for each job (which would be unfortunate, but is plausible). I just want to confirm that you checked this before proposing the change.

Reply:
Very good point, and with all the efforts to save memory left and right, I did not even think about it. Duplicating the entire data array 'n_jobs' times would be very unfortunate. 

But it looks like it was well design:

https://github.com/mne-tools/mne-python/blob/aeeb111f8555b49460583a34c573437358c1cb9f/mne/io/base.py#L950-L965

If I read this correctly, each job receives `data_in[p]` which contains a single channel.

Reply:
But I guess in the end `data_picks_new` is a duplicate of the entire array..

Reply:
wait...  if I'm reading the code you quoted correctly, `data_picks_new` is *one array* that is the same size as `data_in`, but (crucially) it's outside a for-loop over picks.  So that suggests that `n_jobs * n_times` (your original change) was in fact correct? But I'm not intimately familiar with the parallel code, maybe @larsoner can clarify for us.

Reply:
I don't think so. `data_picks_new` is indeed the same shape as `data_in` (except it's a list of `len(picks)` arrays of shape `(1, n_times)` instead of a 2D array). 

So after the execution of `parallel()` we end up with 2 arrays of shape `(len(picks), n_times)`: `self._data` (also called `data_in` but it's the same array) and `data_picks_new`, until garbage collection at the end of the function where the latter is deleted. 

Temporarily, we end up with a second array of shape `(len(picks), n_times)` even if each job receives only one channel (which initially led me to believe the docstring was false).

Reply:
ah, you are right.  I was briefly thinking it was `n_jobs * len(picks) * n_times` (so much worse!) without realizing that I was assuming that. I'll merge then.  Thanks @mscheltienne !